cms/templates/js/video-transcripts.underscore

```diff
@@ -13,14 +13,19 @@
 <% }%>
 <div class='show-video-transcripts-wrapper hidden'>
 <% _.each(transcripts, function(transcriptLanguageText, transcriptLanguageCode){ %>
-    <div id='show-video-transcript-content-<%- transcriptLanguageCode %>' class='show-video-transcript-content'>
+    <% selectedLanguageCodes = _.keys(_.omit(transcripts, transcriptLanguageCode)); %>
+    <div class='show-video-transcript-content' data-edx-video-id="<%- edxVideoID %>" data-language-code="<%- transcriptLanguageCode %>">
+        <div class='transcript-upload-status-container'></div>
         <strong class='transcript-title'><%- StringUtils.interpolate(gettext('{transcriptClientTitle}_{transcriptLanguageCode}.{fileExtension}'), {transcriptClientTitle: transcriptClientTitle, transcriptLanguageCode: transcriptLanguageCode, fileExtension: transcriptDownloadFileFormat}) %></strong>
-        <select id='transcript-language-<%- transcriptLanguageCode %>' class='transcript-language-menu'>
+        <select class='transcript-language-menu'>
             <option value=''>Select Language</option>
             <% _.each(transcriptAvailableLanguages, function(availableLanguage){ %>
-                <option value='<%- availableLanguage[1] %>' <%- transcriptLanguageCode === availableLanguage[1] ? 'selected': '' %>><%- availableLanguage[0] %></option>
+                <% if (!_.contains(selectedLanguageCodes, availableLanguage[1])) { %>
```

Comment:
**Note to reviewers:** This check is for omitting the other server-side saved transcript languages rendering into the dropdown like server side saved are `en, es, de`, this check would restrict to not show `es, de` into `en` selected dropdown and restrict `en and de` for `es` and so on.

Do you think, we should do this? Also does this scenario have any impact when we delete transcript?